cli/src/main/java/com/devonfw/tools/ide/tool/vscode/Vscode.java

```diff
@@ -86,12 +85,16 @@ private void doAddRecommendations(List<PluginDescriptor> recommendations) {
     ObjectMapper objectMapper = new ObjectMapper();
     Map<String, Object> recommendationsMap;
 
+    try (BufferedReader reader = Files.newBufferedReader(extensionsJsonPath)) {
+      recommendationsMap = objectMapper.readValue(reader, Map.class);
+    } catch (IOException e) {
+      throw new RuntimeException(e);
+    }
```

Comment:
You still need to check if the file exists as otherwise this will fail.
```suggestion
  if (Files.exists(extensionsJsonPath)) {
    try (BufferedReader reader = Files.newBufferedReader(extensionsJsonPath)) {
        recommendationsMap = objectMapper.readValue(reader, Map.class);
      } catch (IOException e) {
        throw new RuntimeException(e);
      }
    } else {
      recommendationsMap = new HashMap<>();
    }
```

Reply:
I've Implemented your suggestion now.